### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by nahnahdev**
## In this Branch 

- [ ] Contents of the [Behance page](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) were personalized and the topic chosen was an ECO CONSTRUCTION LANDING PAGE.
- [ ] The home page and The about page were built
- [ ] Each of these pages  have versions for 2 different screen sizes:

-    Mobile: up to 768px wide.
-    Desktop: 768px or wider.

- [ ] The home page has a link in the menu to the about page.
- [ ] The logo in the header links to the home page.
- [ ] In the Mobile menu when the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- [ ] The mobile menu has a close (X) button that closes the menu.
- [ ] The section "Featured speakers" was created dynamically in JavaScript.

- [ ] [Loom Video Link](https://www.loom.com/share/a943838356e544b2ba4e368103d8b989) 
- [ ] [Live Project Link](https://nahnahaj.github.io/construction-page/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,48 @@
...
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
+### Setup Html, Css, and Javascript linters
```
Comment (BertrandConxy) [inline README.md]: - [ ] Good job 👏 👏 Your README looks professional however there are some important points that I would suggest to make it look more professional. The first one is to add the `screenshot` of your landing page. This gives visitors a quick preview of what your site looks like 😃. The second one is to add in the `Getting started` section the procedures to clone and run your project locally 😃 This helps your fellow developers to know how to start with your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  <p>We provide for the daily energy needs of over 2.2 million private and business customers through trading, supply and generation.</p>
+</div>
+<div class="items-lo">
+  <img src="./Images/bio.png">
+<p class="topic">Eco Houses</p>
+<p>We are increasingly offering smart, decentralized solutions in area’s such as smart homes, solar & storage, smart charging etc.</p>
+</div>
+</div>
+<div class="desktop-btn">
+  <button>Join the ECO Family Today!</button>
```
Comment (BertrandConxy) [inline index.html]: - [ ] From the design guidelines, this section is supposed to build 5 individual cards. Currently, you have built 4 cards. so, please add one more 😉 

**Design given**
image

**Your design**
image


- [ ] It is important to note that the button below is styled differently in the desktop version and the mobile version. so please style it accordingly

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+</div>
+  </div>
+</main>
+
+<section id="featured-speakers">
+  <div class="services">
+    <h2>Our Staff</h2>
+    <hr>
+  </div>
+</section>
```
Comment (BertrandConxy) [inline index.html:104]: - [ ] Great job on rendering this section dynamically 👏. In addition to what you have done, I would suggest you change the `background-color` to a `light grey` color in other to match the design given. Please look at the comparison below 👇 

**Design**
image

**Your Design**
image

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @NahnahAJ , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR
- innovative idea
- good UI

## Required Changes ♻️

- [ ] Great job on the video presentation 👏 **the idea behind is awesome and the design looks great**.  However, during the presentation, you forgot **to show and talk about the piece of code that you are proud of which is very important.** This is because it showcases your ability **to explain technical concepts confidently** 😎 Please record another 5-minute video and include that part too 😉 🏃‍♂️ 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,56 +1,58 @@
...
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[ECO BUILDERS](https://nahnahaj.github.io/construction-page/)
 
+> ![](./screenshots/app_screenshot.png)
+> ![](./screenshots/app_screenshot-mobile.png)
 
 ## Getting Started
```
Comment (1GORDON) [inline README.md:22]: Great job @NahnahAJ 🤝

- [ ] [Optional] Just like the previous reviewer pointed out, kindly update the `Getting Started` section on the procedures to clone and run your project locally. This helps your fellow developers to know how to start reading and working on your repo.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @NahnahAJ ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**